configs/config.json

```diff
@@ -8,5 +8,8 @@
     "serverPort": 5000,
     "gasMax": 10,
     "dbFile": "/tellorDB",
+    "paidApis": {
+        "VIXEOD": ["json(https://www.quandl.com/api/v3/datasets/CHRIS/CBOE_VX1.json?api_key=${VIXEOD_KEY}).dataset.data.0.4"]
```

Comment:
note that the name of the key should be "Symbol" + "_KEY" in order for the script to work